adapters/Nend/Public/Headers/GADNendRewardedNetworkExtras.h

```diff
@@ -2,13 +2,11 @@
 //  GADNendRewardedNetworkExtras.h
 //  NendAdapter
 //
-//  Copyright © 2017 F@N Communications. All rights reserved.
+//  Copyright © 2017 FAN Communications. All rights reserved.
 //
 
-#import <GoogleMobileAds/GoogleMobileAds.h>
+#import "GADMAdapterNend.h"
 
-@interface GADNendRewardedNetworkExtras : NSObject<GADAdNetworkExtras>
-
-@property(nonatomic, copy) NSString *userId;
+@interface GADNendRewardedNetworkExtras : GADMAdapterNendExtras
```

Comment:
Can we rename `GADNendRewardedNetworkExtras` to `GADMAdapterNendExtras` since `GADNendRewardedNetworkExtras` doesn't seem to have any implementation?

Reply:
I think we cannot rename `GADNendRewardedNetworkExtras`.
`Google Mobile Ads SDK` seems to bridge only reserved `GADAdNetworkExtras`  subclass name.

(I tried but it's not working.)

Reply:
I'm able to change it on my end.

Make sure to change every instance of `GADNendRewardedNetworkExtras` to `GADMAdapterNendExtras`.